http/router.go

```diff
@@ -12,5 +12,6 @@ func Router(app *gin.Engine) {
 	app.GET(viper.GetString(c.GetAllTasksEndpoint), handler.GetAllTasks)
 	app.GET(viper.GetString(c.GetTaskByIdEndpoint), handler.GetTaskById)
 	app.POST(viper.GetString(c.CreateTaskEndpoint), handler.CreateTask)
-	app.PATCH(viper.GetString(c.UpdateTaskEndpoint), handler.UpdateTask)
+	app.PATCH(viper.GetString(c.UpdateTaskEndpoint), handler.UpdateTaskById)
```

Comment:
Why did you use patch here?

Reply:
Bcz Postgrest uses the patch method for updates. That's why, here I'm using it

Reply:
To my knowledge, both are same. Correct me if I'm wrong.